### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: src/main/java/taxi/filter/AuthenticationFilter.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) request;
+        HttpServletResponse resp = (HttpServletResponse) response;
+        String path = req.getServletPath();
+        if (path.equals("/login") || path.equals("/drivers/add")) {
+            chain.doFilter(req, resp);
+            return;
+        }
+        if (req.getSession().getAttribute("driver_id") == null) {
```
Comment (Max-Pochepets) [inline src/main/java/taxi/filter/AuthenticationFilter.java:34]: TOTHINK: what if logged in driver deletes themselves?

**Review by Max-Pochepets — APPROVED** [review]
👍

File: src/main/java/taxi/filter/AuthenticationFilter.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.service.DriverService;
+
+public class AuthenticationFilter implements Filter {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final DriverService driverService = (DriverService)
```
Comment (RomanDubovskyi) [inline src/main/java/taxi/filter/AuthenticationFilter.java]: do we use it here?

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]
don't forget to fix points, mentioned in the chat

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**